src/app/about-us/page.tsx

```diff
@@ -1,22 +1,27 @@
 import React, { FC } from "react";
 import AboutHero from "@/components/about-us/AboutHero";
-import AboutOurMission from "@/components/about-us/AboutOurMission";
-import BoardMembers from "@/components/about-us/BoardMembers";
-import MissionStatement from "@/components/about-us/MissionStatement";
-import Timeline from "@/components/about-us/Timeline";
+import AboutOurMissionVision from "@/components/about-us/AboutOurMissionVision";
 import missionStatementData from "@/data/about-us";
```

Comment:
**suggestion (non-blocking):** This is a holdover from the old landing site.  It'd be better to remove the data file and hardcode the content in the child components.  This can be done by including the content directly in the react code, or moving the data into an object at the top of the file that is then used by the react code.